### PR TITLE
[Documentation] Fixed typos

### PR DESCRIPTION
**Pull request by pborreli**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | N/A
| License         | MIT

### CODE REVIEW

File: docs/book/promotions.rst [hunk 1]
```diff
@@ -67,7 +67,7 @@ In the example you can see how to create a simple Cart Quantity rule. It will ch
 
    $quantityRule = $ruleFactory->createCartQuantity('5');
 
-   // add your roule to the previously created Promotion
+   // add your route to the previously created Promotion
```
Comment (michalmarcinkowski) [inline docs/book/promotions.rst]: Should be `rule`

Comment (pborreli) [inline docs/book/promotions.rst]: Fixed, thanks

**Comment by pjedrzejewski** [comment]
I have restarted the build - it was failing due to some github outage.

**Comment by pborreli** [comment]
build succeed on PHP7 but has not been restarted on the rest of the matrix

**Comment by michalmarcinkowski** [comment]
Thank you Pascal! 👍

**Pull request merged**